components/desktop/openbox/openbox.p5m

```diff
@@ -0,0 +1,178 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2017 Dariusz Sendkowski 
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/bin/gdm-control
+file path=usr/bin/gnome-panel-control
+file path=usr/bin/obxprop
+file path=usr/bin/openbox
+file path=usr/bin/openbox-gnome-session
+file path=usr/bin/openbox-kde-session
+file path=usr/bin/openbox-session
+file path=usr/etc/xdg/openbox/autostart
+file path=usr/etc/xdg/openbox/environment
+file path=usr/etc/xdg/openbox/menu.xml
+file path=usr/etc/xdg/openbox/rc.xml
+file path=usr/include/openbox/3.6/obrender/color.h
+file path=usr/include/openbox/3.6/obrender/font.h
+file path=usr/include/openbox/3.6/obrender/geom.h
+file path=usr/include/openbox/3.6/obrender/gradient.h
+file path=usr/include/openbox/3.6/obrender/image.h
+file path=usr/include/openbox/3.6/obrender/instance.h
+file path=usr/include/openbox/3.6/obrender/mask.h
+file path=usr/include/openbox/3.6/obrender/render.h
+file path=usr/include/openbox/3.6/obrender/theme.h
+file path=usr/include/openbox/3.6/obrender/version.h
+file path=usr/include/openbox/3.6/obt/display.h
+file path=usr/include/openbox/3.6/obt/keyboard.h
+file path=usr/include/openbox/3.6/obt/link.h
+file path=usr/include/openbox/3.6/obt/paths.h
+file path=usr/include/openbox/3.6/obt/prop.h
+file path=usr/include/openbox/3.6/obt/signal.h
+file path=usr/include/openbox/3.6/obt/util.h
+file path=usr/include/openbox/3.6/obt/version.h
+file path=usr/include/openbox/3.6/obt/xml.h
+file path=usr/include/openbox/3.6/obt/xqueue.h
+link path=usr/lib/$(MACH64)/libobrender.so target=libobrender.so.32.0.0
+link path=usr/lib/$(MACH64)/libobrender.so.32 target=libobrender.so.32.0.0
+file path=usr/lib/$(MACH64)/libobrender.so.32.0.0
+link path=usr/lib/$(MACH64)/libobt.so target=libobt.so.2.0.2
+link path=usr/lib/$(MACH64)/libobt.so.2 target=libobt.so.2.0.2
+file path=usr/lib/$(MACH64)/libobt.so.2.0.2
+file path=usr/lib/$(MACH64)/pkgconfig/obrender-3.5.pc
+file path=usr/lib/$(MACH64)/pkgconfig/obt-3.5.pc
+file path=usr/libexec/openbox-autostart
```

Comment:
For reference, default transforms set the modes at https://github.com/OpenIndiana/oi-userland/blob/oi/hipster/transforms/defaults#L105